internal/store/postgres/migrations/000012_update_status_provider_for_appeal_source.up.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE
+    "grants"
+SET
+    "status_in_provider" = "status"
+WHERE
+    "source" = 'appeal'
+    AND "status_in_provider" != "status"
```

Comment:
if I assume it correctly this is for fixing the previously wrong-assigned data(?) If so,  can we not do it through app migration but run this SQL fix in our running db instances directly?